components/PopupMenu/PopupMenu.js

```diff
@@ -0,0 +1,46 @@
+import styled from "styled-components";
+import { HiDotsVertical } from "react-icons/hi";
+import { useState } from "react";
+
+export default function PopupMenu({ onDelete, id, setPopup }) {
+  return (
+    <>
+      <StyledPopupMenu>
+        <StyledMenuButton onClick={() => onDelete(id)}>delete</StyledMenuButton>
+        <StyledDivider></StyledDivider>
+        <StyledMenuButton onClick={() => setPopup(false)}>
+          close
+        </StyledMenuButton>
+      </StyledPopupMenu>
+    </>
```

Comment:
You don't need a Fragment `<>`for this case because your `<StyledPopupMenu>`is already the top tag:

```suggestion
      <StyledPopupMenu>
        <StyledMenuButton onClick={() => onDelete(id)}>delete</StyledMenuButton>
        <StyledDivider></StyledDivider>
        <StyledMenuButton onClick={() => setPopup(false)}>
          close
        </StyledMenuButton>
      </StyledPopupMenu>
```